packages/perseus/src/types.ts

```diff
@@ -228,6 +228,8 @@ export type APIOptions = Readonly<{
     // This is useful for when we need to know how a widget has shuffled its
     // the available choices.
     onWidgetStartProps?: (widgets: PerseusWidgetMap) => PerseusWidgetMap;
+    // Feature flags that can be passed from consuming application
+    flags?: Record<string, boolean | Record<string, boolean>>;
```

Comment:
I don't know where my comment went. 🙃 

I wonder if we should type this more strictly. Having explicit keys will make integration into the host applications easier as it'll be obvious which flags Perseus supports. It will also help Perseus because we'll only access keys/flags that we know the host knows about. 

So maybe something like this?

```suggestion
    flags?: {
        mafs?: {
            segment: boolean 
        }
    }
```

Reply:
You commented on a resolved conversation, but I saw it!

Reply:
I was really hoping to avoid temporary types that would require cleanup. Index signatures of string to boolean are the usual pattern for feature flags-- it's how it's implemented everywhere else.